service/arm_templates/main.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterID": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique ID of this guest cluster."
+      }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS",
+        "Standard_ZRS",
+        "Premium_LRS"
+      ]
+    },
+    "virtualNetworkCidr": {
+      "type": "string",
+      "metadata": {
+        "description": "The main CIDR block reserved for this virtual network."
+      }
+    },
+    "masterSubnetCidr": {
+      "type": "string"
+    },
+    "workerSubnetCidr": {
+      "type": "string"
+    },
+    "apiLoadBalancerCidr": {
```

Comment:
I like the new names but I have a question on the CIDR's. Do we need to provide these or will they be outputs when the load balancer resources are created?

If we need to provide them then they need to be added to `azuretpr`.

Reply:
Both can work, but I think ultimately they will be provided by the azuretpr.

Reply:
Cool having them in the TPR is fine.